languages/elixir/exercises/concept/recursion/.docs/after.md

```diff
@@ -0,0 +1,72 @@
+Recursive functions are functions that call themselves.
+
+A recursive function needs to have at least one _base case_ and at least one _recursive case_.
+
+A _base case_ returns a value without calling the function again. A _recursive case_ calls the function again, modifying the input so that it will at some point match the base case.
```

Comment:
I'm gonna liberally "borrow" this explanation for the F# exercise. I love this explanation!